Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportWidget.h

```diff
@@ -8,13 +8,15 @@
 #pragma once
 
 #include <QSettings>
+#include <QMouseEvent>
 #include <AtomToolsFramework/Viewport/RenderViewportWidget.h>
 #include <AzFramework/Viewport/CameraInput.h>
 
 #include <EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/RenderPlugin/ViewportPluginBus.h>
 #include <EMStudio/AnimViewportRequestBus.h>
 #include <Integration/Rendering/RenderFlag.h>
 
+
```

Comment:
uber nit: Lose the extra line break

Reply:
I thought we suppose to have double line break between include and actual code starts. The standard is a grey area probably?